src/redux/reducers/components.ts

```diff
@@ -0,0 +1,13 @@
+import initialState, { ReduxState } from '../initialState';
```

Comment:
each of these reducers is concerned w/ only one key in our store and we compose them in the root of this directory w/ the `combineReducers` helper